src/main/java/org/jenkinsci/lib/envinject/service/EnvVarsPipelinedResolver.java

```diff
@@ -0,0 +1,235 @@
+package org.jenkinsci.lib.envinject.service;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.EnvVars;
+import hudson.FilePath;
+import hudson.Util;
+import hudson.model.Action;
+import hudson.model.Computer;
+import hudson.model.Job;
+import hudson.model.Node;
+import hudson.model.Run;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.slaves.NodeProperty;
+import hudson.slaves.NodePropertyDescriptor;
+import hudson.util.DescribableList;
+import jenkins.model.Jenkins;
+import jenkins.security.MasterToSlaveCallable;
+import org.jenkinsci.lib.envinject.EnvInjectException;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.io.Serializable;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Map;
+
+/**
+ * @author Gregory Boissinot
+ * @deprecated The actual version of this API class is located in EnvInject API Plugin
+ */
+@Deprecated
+@Restricted(NoExternalUse.class)
+@SuppressFBWarnings(value = "SE_NO_SERIALVERSIONID", justification = "Deprecated code")
+public class EnvVarsPipelinedResolver implements Serializable {
```

Comment:
https://github.com/jenkinsci/envinject-api-plugin/blob/master/src/main/java/org/jenkinsci/plugins/envinjectapi/util/EnvVarsResolver.java was refactored to support  Pipeline. Looks like there is no practical differences between classes